bot/exts/fun/anagram.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+TIME_LIMIT = 60
+
+# Issue with the json file , gotta try it on local environment first
+ANAGRAMS_ALL = json.loads(Path("bot/resources/fun/anagram.json").read_text("utf8"))
```

Comment:
Don't know if it is worth a change, but would this work for you?
```suggestion
with open(Path("bot/resources/fun/anagram.json")) as f:
    ANAGRAMS_ALL = json.loads(f.read_text("utf8"))
```